ui/src/app/views/cohort-edit/component.spec.ts

```diff
@@ -0,0 +1,267 @@
+import {TestBed, async, fakeAsync, tick} from '@angular/core/testing';
+import {Title, By} from '@angular/platform-browser';
+import {FormsModule} from '@angular/forms';
+import {ActivatedRoute, UrlSegment} from '@angular/router';
+import {CohortEditComponent} from 'app/views/cohort-edit/component';
+import {RouterTestingModule} from '@angular/router/testing';
+import {BehaviorSubject} from 'rxjs/BehaviorSubject';
+import {SignInService} from 'app/services/sign-in.service';
+import {WorkspaceComponent} from 'app/views/workspace/component';
+
+import {Cohort, CohortListResponse} from 'generated';
+import {CohortsService} from 'generated';
+import { Observable } from 'rxjs/Observable';
+
+class Workspace {
+  id: string;
+  namespace: string;
+  cohorts: Cohort[];
+}
+
+function simulateInput(fixture: any, element: any, text: string) {
```

Comment:
Fixture is a typed ComponentFixture. However, it requires type arguments (fixture: ComponentFixture<CohortEditComponent>, which I don't particularly want to give it, because that would restrict this helper to only this test suite.